resources/js/package.js

```diff
@@ -110,6 +110,23 @@ function init() {
                     this.scrollLock = bool
                 }
             },
+            resizedPath(imagePath, size, store = null) {
+                if (!store) {
+                    store = window.config.store
+                }
+
+                let mediaPosition = imagePath.indexOf('/media/')
+                if (mediaPosition > 0) {
+                    return `/storage/${store}/resizes/${size}/magento` + imagePath.substr(mediaPosition + 6)
```

Comment:
This seems deprecated? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr, and it's unclear what we're stripping. What are those 6 characters?